src/comments.js

```diff
@@ -154,7 +154,12 @@ function attach(comments, ast, text) {
 
       if (followingNode) {
         // Always a leading comment.
-        addLeadingComment(followingNode, comment);
+        if (followingNode.type === "Identifier" &&
```

Comment:
Sounds good to me. I'll work on it.

The issue is that

```js
a
  // comment
  .method()
```

looks like

```js
MemberExpression(
  Identifier(a)
    // comment
    Identifier(method)
)
```

in the ast. The AST a user would expect however is

```js
Identifier(a)
// Comment
MemberExpression(Identifier(method))
```